types/stylelint/index.d.ts

```diff
@@ -375,7 +375,23 @@ declare namespace stylelint {
 		cwd: string;
 		results: LintResult[];
 		errored: boolean;
+		/**
+		 * @deprecated Use `report` for the formatted problems, or use `code`
+		 *   for the autofixed code instead. This will be removed in the next major version.
+		 */
 		output: string;
+		/** @internal To show the deprecation warning. */
```

Comment:
suggestion

I am not seeing `@internal` on https://jsdoc.app/
Maybe use https://jsdoc.app/tags-private.html instead?

Reply:
TypeScript uses `@internal`, for example:
https://github.com/microsoft/TypeScript/blob/1a68590c9cd881b23d96424ea49bc9f42e524765/src/services/jsDoc.ts#L183

But the TypeScript document doesn't mention `@internal`:
https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html

I am unsure about this and don't have a strong opinion, but it's off-topic. Can you please open an issue about the usage of `@internal`?

Reply:
related: Microsoft/TypeScript-TmLanguage#480

Should we use [`--stripInternal`](https://www.typescriptlang.org/tsconfig#stripInternal)?